docs/src/content/en/docs/agents/using-tools.mdx

```diff
@@ -41,7 +52,9 @@ export const weatherTool = createTool({
 
 ## Adding tools to an agent
 
-To make a tool available to an agent, add it to the `tools` option and reference it by name in the agent’s instructions.
+To make a tool available to an agent, add it to `tools`. Mentioning available tools and their general purpose in the agent's system prompt will improve tool usage.
```

Comment:
the second sentence makes adding it to `tools` sound optional 

“Improve tool usage” sounds vague - consider something like, help the agent pick the right tool for the job.”